packages/client/src/components/PostDelete.tsx

```diff
@@ -0,0 +1,45 @@
+import { Button, HStack, useColorModeValue } from '@chakra-ui/react';
+import React from 'react';
+import { useNavigate } from 'react-router-dom';
+import { useLocalStorage } from 'react-use';
+import { useDeleteSinglePost } from '../hooks/useGetSinglePost';
+
+interface Props {
+  postId: string;
+}
+
+export const PostDelete = (props: Props) => {
+  const { postId } = props;
+  const [token] = useLocalStorage<string | undefined>('auth');
```

Comment:
why can't this token, based off the name that should be their JWT, why cant you pull this off your `AuthContext`??